tsconfig.json

```diff
@@ -10,7 +10,9 @@
         "moduleResolution": "node",
         "rootDir": "src",
         "module": "commonjs",
-        "declaration": false,
+        "declaration": true,
+        "declarationDir": "./dist/types",
+        "declarationMap":true,
```

Comment:
I could remove the manual type declaration file 👆🏽 because we have auto-generated declarations with maps in the same (dist/types) folder from now. The consumer project remains the same.